Overlays/Windows/cmake/build_options.cmake

```diff
@@ -95,5 +103,28 @@ foreach(type EXE SHARED STATIC)
       set(CMAKE_${type}_LINKER_FLAGS_RELEASE
         "${CMAKE_${type}_LINKER_FLAGS_RELEASE} /OPT:ICF")
     endif()
+
+    # disable incremental linking
+    if(${WN_MSVC_DISABLE_INCREMENTAL_LINKING})
```

Comment:
This should probably be an option (in the root `CMakeLists.txt`)
```
option(WN_MSVC_DISABLE_INCREMENTAL_LINKING "Disable incremental linking (reduces build size at the cost of incremental link time)" ${WN_MSVC_DISABLE_INCREMENTAL_LINKING})
```

This can also look like:
```
if (WN_MSVC_DISABLE_INCREMENTAL_LINKING)
```
CMake will automatically dereference variables like this

Reply:
Updated